src/ripple/protocol/JsonFields.h

```diff
@@ -327,10 +333,12 @@ JSS ( propose_seq );                // out: LedgerPropose
 JSS ( proposers );                  // out: NetworkOPs, LedgerConsensus
 JSS ( protocol );                   // out: PeerImp
 JSS ( pubkey_node );                // out: NetworkOPs
-JSS ( pubkey_validator );           // out: NetworkOPs
+JSS ( pubkey_publisher );           // out: ValidatorList
+JSS ( pubkey_validator );           // out: NetworkOPs, ValidatorList
 JSS ( public_key );                 // out: OverlayImpl, PeerImp, WalletPropose
 JSS ( public_key_hex );             // out: WalletPropose
 JSS ( published_ledger );           // out: NetworkOPs
+JSS ( publisher_lists);             // out: ValidatorList
```

Comment:
nit: `( publisher_lists)` --> `( publisher_lists )`
also for `refresh_interval_min` and `validator_list_expires` below